proto/buoyant-cloud-api.proto

```diff
@@ -130,10 +130,24 @@ message CertificateInfo {
   }
 }
 
+message Server {
+  bytes server = 1;
+}
+
+message ServerAuthorization {
+  bytes server_authorization = 1;
+}
+
+message AuthPolicyInfo {
+  repeated Server servers = 1;
+  repeated ServerAuthorization server_authorizations = 2;
+}
+
 message LinkerdMessage {
   Auth auth = 1;
   oneof message {
     CertificateInfo crt_info = 2;
+    AuthPolicyInfo auth_policy_info = 3;
```

Comment:
i just remembered we'll want support for `trafficsplits.split.smi-spec.io` and `serviceprofiles.linkerd.io` as well.

Reply:
that is correct, do we want to bundle all messages under one type that packs the Linkerd CRDs ?

Reply:
how about we keep AuthPolicyInfo as-is, since they're quite related. then maybe add two more for the others...
```proto
  oneof message {
    CertificateInfo crt_info = 2;
    AuthPolicyInfo auth_policy_info = 3;
    ServiceProfile
    TrafficSplit
```
wdyt?

Reply:
Makes sense, we might also want to throw in the multicluster CRDs... Also.. I think we are facing a similar problem with these types as we are with the auth CRDs. Although we have clients for them, we do not have proto representations so we will still need to rely on JSON serialization. FWIW, all types in the k8s API have proto representations that have been generated automatically by a code-gen tool they use: https://github.com/kubernetes/api/blob/master/core/v1/generated.proto

Reply:
ah, i missed the multi-cluster CRDs, i guess those come from a separate linkerd extension?

re: serializing these structs. i thought that we could instantiate serializers to handle our custom CRDs. we do related things in linkerd when converting to/from various types:
https://github.com/linkerd/linkerd2/blob/main/pkg/k8s/fake.go#L172-L193